.github/blueprints/blueprint.json

```diff
@@ -0,0 +1,23 @@
+{
+  "$schema": "https://playground.wordpress.net/blueprint-schema.json",
+  "landingPage": "/wp-admin/plugins.php",
```

Comment:
I think it would be ideal to send them to the sitemap itself (so `/news-sitemap.xml`) instead of the plugin screen. In addition, we may want a step here that imports some test content so there's things to actually show in that sitemap